From a57f7a2eca3061f1392bd0490584b6a2a1913784 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Fri, 13 Dec 2013 14:42:59 +0000
Subject: [PATCH 138/462] bugfix, extension Mantis 0029957: ConveyorComp:
 CUSS1.3 Conveyor (without ConveyorAEA)

AntTargets.xml
  use GenXMLSrcDir for generated source files from xml schema files
  fixed clean; delete GenXMLSrcDir and TestClassDir


git-svn-id: svn://localhost/SelfServiceBuild/trunk@200 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntTargets.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/SelfServiceBuild/Build/java/AntTargets.xml b/SelfServiceBuild/Build/java/AntTargets.xml
index 376bd56..6e96010 100644
--- a/SelfServiceBuild/Build/java/AntTargets.xml
+++ b/SelfServiceBuild/Build/java/AntTargets.xml
@@ -257,7 +257,9 @@
 
     <!-- Die erzeugten Unterverzeichnisse werden geloescht. -->
     <delete dir="${GenSrcDir}"/>
+    <delete dir="${GenXMLSrcDir}"/>    
     <delete dir="${ClassDir}"/>
+    <delete dir="${TestClassDir}"/>    
     <delete dir="${ManifestDir}"/>
     <delete dir="${JavaDocDir}/${ant.project.name}"/>
 
-- 
2.41.0.windows.1

